google/resource_binaryauthorization_attestor.go

```diff
@@ -0,0 +1,398 @@
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package google
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBinaryauthorizationAttestor() *schema.Resource {
```

Comment:
Can we fix this capitalization?  I did it with a hack for ResourceManager iirc.

Reply:
Yup! I fixed the hack too.